content/productivity/crawl-planets/index.md

```diff
@@ -2,5 +2,5 @@
 title: Crawl Planets
 date: 2021-07-29T14:00:00+02:00
 subtitle: Capture unowned planets around you!
-version: 0.6.0
+version: 0.6.1
```

Comment:
This is more of a game version thing. Were mostly leaving these at 0.6.0 for the v6 round
```suggestion
version: 0.6.0
```